src/components/FeatureViews/Accounts/TabComponents/EnrollmentView.js

```diff
@@ -383,21 +382,25 @@ const CourseSessionStatus = () => {
                         </Typography>
                     </Grid>
                     <Grid item xs={3}>
-                        <FormControl component="fieldset">
-                            <FormGroup>
-                                <FormControlLabel
-                                    control={
-                                        <Switch
-                                            checked={highlightSession}
-                                            onChange={handleHighlightSwitch}
-                                            color="primary"
-                                            value="upcoming-session"
+                        <Grid container alignItems="flex-start">
+                            <Grid item>
```

Comment:
Is it possible to reduce the triple nested `<Grid>`s? (I think the outer two levels can be combined into one tag)

Reply:
whaaaaoooo didn't know this was a thing for mui3.0 :o